src/splitAt.js

```diff
@@ -0,0 +1,25 @@
+var _curry2 = require('./internal/_curry2');
+var length = require('./length');
+var slice = require('./slice');
+
+
+/**
+ * Splits a given list or string at a given index.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig Number -> [a] -> [[a], [a]]
+ * @sig Number -> String -> [String, String]
+ * @param {Number} a
+ * @param {Array|String} b
```

Comment:
I believe JSDoc expects a parameter name between `@param` and `{…}`. See one of the other files in __src__ directory for an example.